MAUI/NumericEntry/UpDown-Button.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: Use UpDown Button in .NET MAUI NumericEntry | Syncfusion
+description: Learn here all about how to use UpDown Button (SpinButton) in Syncfusion .NET MAUI NumericEntry (SfNumericEntry) control and more.
+platform: MAUI
+control:  SfNumericEntry
+documentation: ug
+---
+
+# UpDown Button in .NET MAUI NumericEntry
+
+This section describes how to change value in the `NumericEntry` control using keys, mouse scrolling and the up-down button.
+
+## Increase or decrease value
+
+You can increment or decrement the value in the `NumericEntry` control by using **UpArrow**, **DownArrow**, **PageUp** and **PageDown** keys. You can change the increment or decrement value when Arrow keys are pressed, using the SmallChange property and for Page keys using the LargeChange property. By default, the value of `SmallChange` property is **1** and `LargeChange` property  is **10**.
+
+N> The value in `NumericEntry` can also be changed by mouse scrolling. The mouse scrolling increases or decreases the value based on the `SmallChange` property.
```

Comment:
N> The value in the `NumericEntry` can also be changed by mouse scrolling. The mouse scrolling increases or decreases the value based on the `SmallChange` property.

Reply:
The requested content modification has been successfully addressed